source/docs/being-a-team/how-to-ask-effective-questions.rst

```diff
@@ -0,0 +1,69 @@
+
+Research First
+--------------
+
+Before you ask a question, you should be doing your own research to find the answer.
+This will take more time, it will take more effort, but it is a crucial skill to learn to find solutions on your own, and learn how to research effectively to find solutions to your problems.
+In the end, this research will pay off in the long run, helping you to learn about topics relevant to your question, and often helping you to understand the roots of the problems you encounter.
+
+Here are some steps that you could take to research your problem before asking someone a question about your current problem.
+
+- **Google** - Simply search your problem.
+  You will be surprised by how often this can quickly solve your problem although you may not have expected it to.
+  Try word the question in many different ways to find similar questions that may have already been asked.
+
+.. tip::
+    Add "filetype:pdf" to the beginning of your search to filter the search results into only .pdf documents.
+    This can help you when looking for past documents or sections of textbooks.
+
+- **Youtube** - Just like google, there are many great resources on Youtube which can help with very specific, obscure problems.
+- **Reddit and Discord** - If you can't find your own question online, the FTC Reddit and Discord will be the other most likely place to have a past question similar to your own.
+  Try searching through the past questions with words related to your question and see if a similar question has been asiked before.
+  You may find information in this discussion which can help you to solve your own problem.
+- **Past Examples** - Try find past implementations of whatever problem you are trying to solve.
+  This step may take more time to find the example, and then decipher how someone else figured out the solution, but this process can be super valuable.
+  For programming, digging through another person's repo can be very insightful, and likewise, for mechanical design, taking a look at a teams technical binder or CAD release.
+
+Be Specific
+-----------
+
+For others to be able to provide a good solution to your problem, they need the specifics.
+This means doing the following:
+
+1. **Break down the problem.** You should try to pinpoint the problem as much as possible.
+For example, when encountering robot crashes, try to collect as much data as possible around the times when your robot crashes.
+You may not understand this information yourself, but it can be a huge help to others trying to diagnose your issue.
+
+2. **Provide some context.** It is often helpful to provide context for example,  what skills you are new to and hence, could be the root of the issues.
+Furthermore, information about your resources can help others to determine solutions which are feasible and effective within your own constraints.
```

Comment:
```suggestion
Furthermore, information about your resources can help others to determine solutions which are feasible and effective within your own constraints.
For example, you might be competing within a week and looking for a quick fix; make sure to mention this in your question if so.

It can also be useful to explain why you're asking. It can be easy to get stuck on a specific solution to a problem when it would be better solved in an entirely different way; this is known as the [XY problem.](https://xyproblem.info/)
```
Here's a start on a mention of XY problem, not sure whether this is as clear as it could be or in the right spot though.

Reply:
Added to step 1, break down the problem.